test/integration/hotrestart_main.cc

```diff
@@ -0,0 +1,58 @@
+#include "common/stats/utility.h"
+
+#include "exe/main_common.h"
+
+#include "absl/debugging/symbolize.h"
+
+// NOLINT(namespace-envoy)
+
+/**
+ * Custom main() for hotrestart_test. This should be identical to
+ * source/exe/main.cc, except for the registration and increment of a new gauge
+ * specifically for hot_restart.test.sh.
+ *
+ * If this test starts failing to run or build, please ensure that all changes
+ * from source/exe/main.cc are incorporated in this file.
+ *
+ * TODO(jmarantz): factor out this code into MainCommon to avoid duplication,
+ * noting that absl::InitializeSymbolizer may want to be directly by main().
```

Comment:
```suggestion
 * noting that absl::InitializeSymbolizer may want to be directly called by main().
```

Reply:
I've gotten confirmation from the absl team that running InitializeSymbolizer in a helper is OK, so I went ahead and did the TOOD and remove the comment you fixed here.